secretpolaris.shopify.com/dev.yml

```diff
@@ -0,0 +1,12 @@
+name: polaris-shopify-com
```

Comment:
dev doesn't really like sub folders. you should have a dev.yml at the root of your repository not on a per sub-folder basis